zerver/openapi/zulip.yaml

```diff
@@ -795,6 +910,20 @@ components:
           code:
             type: string
             description: A string that identifies the error.
+    BadEventQueueIdError:
+      allOf:
+      - $ref: '#/components/schemas/CodedError'
+      - properties:
+          queue_id:
+            type: string
+            description: The string that identifies the invalid event queue.
+      - example:
+          {
+              "code": "BAD_EVENT_QUEUE_ID",
+              "msg": "Bad evetn queue id: 1518820930:1",
```

Comment:
This has a typo in "event"?  Typos in this sort of thing that isn't checked by automated tests can be very hard to catch and problematic, so I highly recommend using copy/paste to move around API-related details in your editor to avoid them.

Reply:
99% of times I just copy-paste from the client I use to interact with the API, so I'm not sure how could that have happened. I'll review those more carefully in the future, just in case a random character slips in or something.